services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -112,6 +112,7 @@ async def get_containers_inactivity(
         ContainerExecContainerNotFoundError,
         ContainerExecCommandFailedError,
         ContainerExecTimeoutError,
+        json.JSONDecodeError,
```

Comment:
Would it make sense to write a different message in the logs depending on these cases, so that it is easier to go in and debug when it happens?

Reply:
I think it's fine. It's used to give an idea about what happened. At most I can reword it to make it sound more generic.

Reply:
I think you have a good point now that I think about it. Let me make it a bit better.

Reply:
I think something like this would look much better
```
Could not parse command result 'This is an unparsable json response {}' as 'InactivityResponse'
Traceback (most recent call last):
  File "/home/silenthk/work/pr-osparc-fix-inactivity-error/services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py", line 126, in get_containers_inactivity
    return parse_raw_as(InactivityResponse, inactivity_response)
  File "pydantic/tools.py", line 74, in pydantic.tools.parse_raw_as
  File "pydantic/parse.py", line 37, in pydantic.parse.load_str_bytes
  File "/home/silenthk/.pyenv/versions/3.10.4/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/home/silenthk/.pyenv/versions/3.10.4/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/silenthk/.pyenv/versions/3.10.4/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```